calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,10 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --enable-docker-networking-labeling
```

Comment:
I think the doc string here needs a little tweak.  This doesn't enable adding labels to Docker containers, it enables support for pulling Docker labels from the container and adding those labels to the Calico endpoints for use with Calico policy.

I think we could also say that only "calico.blah" (I can't remember the syntax) namespaced labels are transferred.